### PR TITLE
Add Sublime Text setup instructions

### PR DESCRIPTION
**Pull request by bradJM**
Adds instructions for setting up the LSP for Sublime Text package and adding Solargraph to its list of clients.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,4 +118,38 @@ VSCode supports running solargraph through an [extension](https://marketplace.vi
...
+      "selector": "source.ruby",
+      "initializationOptions": {
+        "diagnostics": true
+      }
+    }
+  }
+}
+```
+
+The LSP package supports automatic formatting but doesn't enable it by default. If you'd like to use it:
```
Comment (bradJM) [inline README.md:145]: There is one small wrinkle to using this feature, but it isn't specific to Sublime Text so this felt like the wrong spot to add it: Solargraph uses Rubocop for formatting (I think, anyway?) and if you're running a newer version of Ruby then Rubocop's formatter will make changes that don't work with DragonRuby's version of mruby. The workaround I've been using is adding these lines to my `.rubocop.yml` file:

```yaml
AllCops:
  TargetRubyVersion: 2.7
  
Style/RandomWithOffset:
  Enabled: false
```

I can add that somewhere else in the README, but is it getting too much into Solargraph config details that people should go to their docs for?

File: README.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ $ git clone git@github.com:owenbutler/dragonruby-yard-doc.git
 
 ### A CRuby runtime
 
-Dragonruby itself does not require a ruby runtime to work. So, you might be wondering why we need another Ruby installed here? The answer is that the solargraph gem, which we will us as an LSP, does need a ruby runtime.
+Dragonruby itself does not require a ruby runtime to work. So, you might be wondering why we need another Ruby installed here? The answer is that the solargraph gem, which we will use as an LSP, does need a ruby runtime.
```
Comment (owenbutler) [inline README.md:57]: ❤️

**Review by owenbutler — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (owenbutler) [reply README.md:145]: I'm inclined to leave it out for now - lets see if anyone else gets snagged by it.

My gut tells me that someone who enables this is proficient enough in Rubocop to know how to do it.

I appreciate the attention to detail though, thanks @bradJM

Comment (bradJM) [reply README.md:145]: Sounds good!

**Pull request merged**